modules/virt-creating-localnet-nad-cli.adoc

```diff
@@ -12,10 +12,33 @@ You can create a network attachment definition (NAD) which describes how to atta
 * You have access to the cluster as a user with `cluster-admin` privileges.
 * You have installed the OpenShift CLI (`oc`).
 * You have installed the Kubernetes NMState Operator.
-* You have created a `NodeNetworkConfigurationPolicy` object to map the OVN-Kubernetes secondary network to an Open vSwitch (OVS) bridge.
 
 .Procedure
 
+. Create a `NodeNetworkConfigurationPolicy` object to map the OVN-Kubernetes secondary network to an Open vSwitch (OVS) bridge:
++
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: mapping # <1>
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/worker: '' # <2>
+  desiredState:
+    ovn:
+      bridge-mappings:
+      - localnet: localnet-network # <3>
+        bridge: br-ex # <4>
+        state: present # <5>
+----
+<1> The name of the configuration object.
+<2> Specifies the nodes to which the node network configuration policy is to be applied.
```

Comment:
FWIW, the node selector is not required. It would apply to all nodes (including control plane).

Maybe we should indicate the provided `nodeSelector` is recommended, since having the underlay configured in the control plane nodes is useless.

Reply:
I would keep the `nodeSelector`. The [VM example later in this doc](https://84261--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.html#virt-attaching-vm-to-ovn-secondary-nw-cli_virt-connecting-vm-to-ovn-secondary-network) doesn't specify any `nodeSelector`, which means that the VM can be scheduled on any node that has the bridge - including control planes/masters, which is not recommended.
This is in addition to the point that Miguel raised - configuring this bridge on the control planes is redundant.

Reply:
@maiqueb and @yossisegev I have updated the callout description highlighted above and the example VM YAML.